src/main/java/ru/skypro/homework/dto/comment/Comment.java

```diff
@@ -0,0 +1,13 @@
+package ru.skypro.homework.dto.comment;
+
+import lombok.Data;
+
+@Data
+public class Comment {
+    private int author;
+    private String authorFirstName;
+    private int createdAt;
```

Comment:
```suggestion
    private long createdAt;
```
int32 - int
int64 - long